doc/nrf/glossary.rst

```diff
@@ -0,0 +1,447 @@
+
+Hardware Flow Control (HWFC)
+   A handshaking mechanism used to prevent a buffer overflow in the receiver (in embedded computing use cases).
+   When the transmission baud rate is high enough for data to appear faster than it can be processed by the receiver, the communicating devices can synchronize with each other, using RTS and CTS pins.
+
+Host Controller Interface (HCI)
+   Standardized communication between the host stack (e.g., a PC or mobile phone OS) and the controller (the Bluetooth IC).
+   This standard allows the host stack or controller IC to be swapped with minimal adaptation.
+
+Human Interface Device (HID)
+   Type of a computer device that interacts directly with, and most often takes input from, humans and may deliver output to humans.
+   The term "HID" most commonly refers to the USB-HID specification.
+   This standard allows the host stack or controller IC to be swapped with minimal adaptation.
+
+Integrated Circuit (IC)
+   A semiconductor chip consisting of fabricated transistors, resistors, and capacitors.
+
+Integrated Development Environment (IDE)
+   A software application that provides facilities for software development.
+
+Internet Control Message Protocol (ICMP)
+   The control protocol of the IP stack that enables the establishment of reachability, routes, and so on.
+   This protocol is an integral part of any IP but is unique as it is not a transport protocol to exchange data between hosts.
+
+Internet Protocol version 4 (IPv4)
+   The fourth version in the development of the Internet Protocol (IP).
+   It is the communications protocol that provides an identification and location system for computers on networks.
+   It routes most traffic on the Internet.
+
+Internet Protocol version 6 (IPv6)
+   The latest version of the Internet Protocol (IP).
+   It is the communications protocol that provides an identification and location system for computers on networks and routes traffic across the Internet.
+
+IPv4 address
+   A numerical label that is used to identify a network interface of a computer or other network node participating in an IPv4 computer network.
+
+IPv6 address
+   An alphanumerical label that is used to identify a network interface of a computer or other network node participating in an IPv6 computer network.
+
+Kconfig file
+   A configuration file for a module, written in the Kconfig language syntax.
+   It defines build-time configuration options, also called symbols, namely application-specific values for one or more kernel configuration options.
+   It also defines how they are grouped into menus and sub-menus, and dependencies between them that determine what configurations are valid.
+
+Kconfig fragment
+   A configuration file used for building an application image with or without software support from specific Kconfig options.
+   Examples include whether to add networking support, which drivers are needed by the application, etc.
```

Comment:
We have another etc here. Please get rid of it.

Reply:
fixed